package.json

```diff
@@ -29,13 +29,15 @@
     "babel-plugin-transform-object-rest-spread": "^6.19.0",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
+    "create-react-class": "^15.6.0",
     "enzyme": "^2.6.0",
     "happiness": "^7.1.2",
     "mocha": "^3.2.0",
     "mochify": "^2.18.1",
-    "react": "<15.5.0",
-    "react-addons-test-utils": "<15.5.0",
-    "react-dom": "<15.5.0"
+    "prop-types": "^15.5.10",
```

Comment:
I believe this should match the react version specified in the peer deps.  Not sure if it makes a difference, but I believe the publish these in lock step.

Reply:
The latest version of prop-types is 15.5.10 so I guess it's diverged from react in that regard.